nebula/ui/components/VPNCheckBox.qml

```diff
@@ -44,11 +43,14 @@ CheckBox {
 
     }
     Keys.onReturnPressed: checkBox.clicked()
-    // TODO
-    // Accessible.name: accessibleName
+    Keys.onSpacePressed: checkBox.clicked()
+
     Accessible.onPressAction: clicked()
     Accessible.onToggleAction: clicked()
     Accessible.focusable: true
+    Accessible.name: accessibleName
+    Accessible.description: accessibleDescription
```

Comment:
same here. I believe the description overwrites the name. Is that intended?

Reply:
Interesting! For voiceover, you are correct- the description overwrites the name but here we are on mac with 'hover text' enabled and the description is not overwritten.

https://user-images.githubusercontent.com/22355127/219451907-94b09782-b54e-4c30-846f-ddebbd82ed0f.mov

Reply:
Removed Accessible.description in favor of Accessible.name in the interest of optimizing for screen readers and brevity but this is a short term solution. Also considered doing something like `${Accesible.name} - ${Accessible.Description}`, but feels risky with l10n.